backend/daemon/api/documents/v1alpha/documents.go

```diff
@@ -31,21 +36,31 @@ type Discoverer interface {
 	Connect(context.Context, peer.AddrInfo) error
 }
 
+// GatewayClient used to connect to the gateway and push content.
+type GatewayClient interface {
+	// GatewayClient used to connect to the gateway and push content.
+	GatewayClient(context.Context, string) (mttnet.GatewayClient, error)
+}
+
 // Server implements DocumentsServer gRPC API.
 type Server struct {
-	db    *sqlitex.Pool
-	me    *future.ReadOnly[core.Identity]
-	disc  Discoverer
-	blobs *hyper.Storage
+	db       *sqlitex.Pool
+	me       *future.ReadOnly[core.Identity]
+	disc     Discoverer
+	blobs    *hyper.Storage
+	gwClient GatewayClient
+	testnet  bool
 }
 
 // NewServer creates a new RPC handler.
-func NewServer(me *future.ReadOnly[core.Identity], db *sqlitex.Pool, disc Discoverer, LogLevel string) *Server {
+func NewServer(me *future.ReadOnly[core.Identity], db *sqlitex.Pool, disc Discoverer, gwClient GatewayClient, LogLevel string, testnet bool) *Server {
```

Comment:
@juligasa This looks to be unused now, right?

Reply:
right, done!